src/main/java/commands/SearchCommand.java

```diff
@@ -50,21 +81,89 @@ public SearchCommand(String userInput, UI ui, PlanList planList,
         LogHandler.linkToFileLogger(logger);
     }
 
+    /**
+     * Constructs a new instance of the SearchCommand. Constructed when the user enters a
+     *  search command for all types.
```

Comment:
a very minor formatting issue: extra whitespace on second line

Reply:
Thanks for finding it for me! even gradle didn't notice haha.